docs/patching_and_chromium_src.md

```diff
@@ -10,6 +10,45 @@ If changes can be made inside existing subclasses and code inside `src/brave`, t
 
 ## Changes inside Chromium
 
+### Minimize dependencies
```

Comment:
this section should be moved down, after introduction to chromium_src.

Reply:
Maybe it should move down below both then. I'll create another PR

Reply:
https://github.com/brave/brave-core/pull/32512